CONTRIBUTOR_LADDER.md

```diff
@@ -60,19 +91,25 @@ A Maintainer must meet the rights, responsiblities, and requirements of a Contri
     * Determining strategy and policy for the project
     * Participating in, and leading, community meetings
     * Helps run the project infrastructure
-* Requirements
-    * Experience as a Contributor for at least 3 months
-    * Demonstrates a broad knowledge of the project across multiple areas
+* Requirements:
+    * Experience as a Reviewer for at least 3 months
+    * Deep understanding of the technical goals and direction of the project
+    * Deep understanding of the technical domain of the project across multiple areas
+    * Sustained contributions to design and direction by doing all of:
+      * Authoring and reviewing proposals (GitHub Issues for refactoring, enhancements, or new functionality)
+      * Initiating, contributing, and resolving discussions (emails, GitHub issues, meetings)
+      * Identifying subtle or complex issues in designs and implementation PRs
     * Is able to exercise judgement for the good of the project, independent of their employer, social circles, or teams
-    * Mentors other Contributors
+    * Mentors other Contributors and Reviewers
 * Additional privileges:
+    * Approve PRs to any area of the project
     * Represent the project in public as a Maintainer
     * Communicate with the CNCF on behalf of the project
     * Have a vote in Maintainer decisions
 
-Process of becoming a maintainer:
+Process of becoming a Maintainer:
 
-1. Any current Maintainer may nominate a current Contributor to become a new Maintainer, by opening a PR against the root of the [OSM repository](https://github.com/openservicemesh/osm) and adding the nominee to the [OWNERS file](https://github.com/openservicemesh/osm/blob/main/OWNERS). Additionally, if the nomination is for a code owner whose PR approvals are meant to satisfy the PR merge requirements, the nominee must be added to the [CODEOWNERS file](https://github.com/openservicemesh/osm/blob/main/CODEOWNERS) in the same PR.
+1. A current Reviewer may be self-nominated or be nominated by a current Maintainer by opening a PR against the root of the [OSM repository] and adding the nominee to the [OWNERS file] under the *owners* entry. Additionally, if the nomination is for a code owner whose PR approvals are meant to satisfy the PR merge requirements, the nominee must be added to the [CODEOWNERS file] in the same PR.
```

Comment:
could you clarify the difference between codeowner and owner? I think explicitly defining codeowners in the first section of the doc could be helpful since it's a common subpart of being a reviewer and a maintainer. i.e., This is our contributor ladder [...] and also we define a codeowner as someone who [...]. This may be common gh terminology but defining it for the scope of our project would help